docs/en/latest/getting-started.md

```diff
@@ -270,3 +272,27 @@ ext-plugin:
 APISIX will treat the plugin runner as a child process of its own, managing its entire lifecycle.
 
 APISIX will automatically assign a unix socket address for the runner to listen to when it starts. environment variables do not need to be set manually.
+
+#### Setting up in container
```

Comment:
To be honest, we don't suggest people use the go-runner binary directly. That binary is used as an example.
What about adding a note for this and removing the `make build` command?

Reply:
@spacewander We need to build binaries, and perhaps we can provide an example of a Dockerfile that produces binaries directly at Docker container image build time to avoid more problems with libc, etc.
I will try it in a later PR.